docs/tutorials/extending_admin_ui/5_creating_a_udw_tab.md

```diff
@@ -0,0 +1,536 @@
+}
+
+.c-image:before {
+    content: attr(data-title);
+    display: flex;
+    background: rgba(0,0,0,.75);
+    color: #fff;
+    width: 300px;
+    align-items: center;
+    justify-content: center;
+    font-weight: 700;
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    opacity: 0;
+    padding: 1rem;
+    transition: opacity .3s ease-in-out;
+    overflow: hidden;
+}
+
+.c-image:hover:before,
+.c-image:focus:before {
+    opacity: 1;
+}
+
+.c-image__thumb {
+    display: block;
+    max-width: 300px;
+    max-height: 200px;
+    width: auto;
+    height: auto;
+    margin: auto;
+}
+```
+
+#### `images.list.css`
+
+``` css
+.c-images-list {
+    display: grid;
+    grid-template-areas: 'prev list next';
+    grid-template-columns: 32px 1fr 32px;
+    grid-gap: 16px;
+    overflow: hidden;
+}
+
+.c-images-list__items-wrapper {
+    overflow: hidden;
+    max-width: 1564px;
+}
+
+[class*="c-images-list__btn--"] {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background: #f15a10;
+    transition: background .2s ease-in-out, opacity .2s ease-in-out;
+}
+
+[class*="c-images-list__btn--"]:focus,
+[class*="c-images-list__btn--"]:hover {
+    background: #ab3f0a;
+}
+
+[class*="c-images-list__btn--"][disabled],
+[class*="c-images-list__btn--"][disabled]:focus,
+[class*="c-images-list__btn--"][disabled]:hover {
+    background: #f15a10;
+    opacity: .5;
+}
+
+[class*="c-images-list__btn--"] .ez-icon {
+    fill: #fff;
+}
+
+.c-images-list__btn--prev {
+    grid-area: prev;
+}
+
+.c-images-list__btn--next {
+    grid-area: next;
+}
+
+.c-images-list__items {
+    grid-area: list;
+    display: flex;
+    flex-wrap: nowrap;
+    transition: transform .3s ease-in-out;
+}
+
+.c-images-list__items .c-image {
+    flex: 0 0 300px;
+}
+
+.c-images-list__items .c-image + .c-image {
+    margin-left: 1rem;
+}
+```
+
+## Compile the files
+
+Now you need to make the React module available (in umd version) in the `window` object in the browser.
+
+Place the compiled files in `src/EzSystems/ExtendingTutorialBundle/Resources/public/js/modules`.
+
+## Add configuration
+
+Next, add the following configuration blocks to `src/EzSystems/ExtendingTutorialBundle/Resources/config/services.yml`:
+
+``` yaml
+ezplatform.udw.image.panel.module.js:
+    parent: EzSystems\EzPlatformAdminUi\Component\ScriptComponent
+    arguments:
+        $src: '/bundles/acmetutorial/js/modules/ImagesPanel.module.js'
+    tags:
+        - { name: ezplatform.admin_ui.component, group: 'custom-admin-ui-modules' }
+
+ezplatform.udw.add.tab.js:
+    parent: EzSystems\EzPlatformAdminUi\Component\ScriptComponent
+    arguments:
+        $src: '/bundles/acmetutorial/js/add.tab.js'
+    tags:
+        - { name: ezplatform.admin_ui.component, group: 'custom-admin-ui-config' }
+```
+
+Finally, make sure that the configuration is imported. In `src/EzSystems/ExtendingTutorialBundle/DependencyInjection` add the following files:
```

Comment:
[...]make sure that the Symfony bundle configuration is imported (we're using a default configuration generated by the output of `php bin/console generate:bundle` command)[...]